### PR TITLE
WRN-10744: Fixed VideoPlayer to return the current state of the media

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed VideoPlayer to return the current state of the media when handling events (e.g., `handlePause`, `handlePlay`)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed `videoPlayerBase.getMediaState()` to return values of `this.video` instead of `this.state`
Fixed event handler(e.g., `handlePause()`) to pass the event to the function passed in prop after the custom event handler is called

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-10744

### CODE REVIEW

File: VideoPlayer/VideoPlayer.js [hunk 1]
```diff
@@ -1331,10 +1393,11 @@ const VideoPlayerBase = class extends Component {
 		this.send('play');
 		this.announce($L('Play'));
 		this.startDelayedMiniFeedbackHide(5000);
+		return true;
```
[the remark this answers is not in the document]
Comment (0x64) [reply VideoPlayer/VideoPlayer.js:1396]: And, could you please add an empty line above the newly added lines for better readability? :)

**Comment by 0x64** [comment]
In _rewind_ function, the main logic of the function is simply passed by _return_ clause if the current time is 0. In this case, should be return _true_ or _false_?

File: VideoPlayer/VideoPlayer.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (juwonjeong) [reply VideoPlayer/VideoPlayer.js:1397]: It's fixed.

**Review by 0x64 — APPROVED** [review]
👍 
needs to be synced with `develop` before merging.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - Support color customization
 
+### Fixed
```
Comment (jeonghee27) [inline CHANGELOG.md:9]: @0x64 
Dosen't we need new "added" changelog for will~ ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
...
+
+- `sandstone/VideoPlayer` to return the current state of the media when handling event and not to invoke the mediaControl related callback when video source is unavailable
```
Comment (jeonghee27) [inline CHANGELOG.md:11]: Let's shorten this...

```suggestion
- `sandstone/VideoPlayer` to handle mediaControl related callbacks properly
```

Comment (jeonghee27) [reply CHANGELOG.md:11]: And Add a "added" changelog



- `sandstone/VideoPlayer` onWillPause, , ,,  , , props

Comment (juwonjeong) [reply CHANGELOG.md]: It's fixed.

**Pull request merged**